src/main/webapp/app/service/firebase/firebase-meta-service.ts

```diff
@@ -73,7 +73,7 @@ export class FirebaseMetaService {
     return Promise.resolve();
   };
 
-  updateGeneCurrentReviewer = async (hugoSymbol: string, isGermline: boolean, isReviewing: boolean) => {
+  updateGeneCurrentReviewer = async (hugoSymbol: string, isGermline: boolean | undefined, isReviewing: boolean) => {
```

Comment:
I don't expect `isGermline` to be undefined

Reply:
Let's revisit this comment after I fix the database types